### DISCUSSION TITLE
Move `About` to hamburger menu

### DISCUSSION THREAD

**Discussion by foss-** [organicmaps/organicmaps#6411] [Ideas]
While I understand OM wants to push Donations, I am not sure making the About button be the most prominent button on the main app view. It is the only colored button and despite not really serving any functionality is listed as first icon on the action icons at the bottom:

<img src="https://github.com/organicmaps/organicmaps/assets/5708172/a652fd92-1d4b-4319-9d85-45ceebaf2b17" width="250">

Adding `About` to the hamburger menu, would be, were that option actually belongs. That would also help de-clutter the main view and only provide the most important icons at that location.

**Comment by biodranik** [comment]
It was already discussed many times. Donations are not the only/main reason. Another important reason for this button is to _teach new users_ about OSM, about app philosophy (it's free, donations-based, run by enthusiasts), and to easily report bugs/issues for new users while avoiding negative reviews in stores.

One more reason is to unobtrusively announce the rare but most important news for the project to all users.

We'll consider improving the situation in the future for _loyal_ and _mature_ users. But now let's keep it simple and efficient.

**Reply by foss-** [reply]
Intro to OSM could be a tutorial.

News could be shown after app was updated.

**Reply by biodranik** [reply]
Most users skip boring tutorials to get to the main functionality faster. Showing unavoidable dialog is often more annoying than showing a red dot/number on the button/icon that can be pressed only when necessary.

**Comment by rtsisyk** [comment]
Thanks for running this poll. Let's discuss idea about removing the bottom menu altogether. I am closing this poll as we are going to phase out GitHub Discussions in favor of GitHub Issues.
